app/resources/add_note.html.template

```diff
@@ -18,35 +18,116 @@
 {% endif %}
 
 <div class="fields-table note" role="complementary">
-  <h2>
-    {% if not markdup %}
-      {% trans "Tell us the status of this person" %}
-    {% else %}
-      {% trans "Mark records as duplicate" %}
-    {% endif %}
-  </h2>
+  <div class="header section">
+    <h1>
+      {% if not markdup %}
+        {% trans "Status of " %}{{person.full_name}}
```

Comment:
This part should be instead:

    {% blocktrans %}Status of {{person.full_name}}{% endblocktrans %}

Because it may be in different order in different languages.

Also, fall back to "Status of the person" if person.full_name is missing. This can happen when you create a new person record:
https://screenshot.googleplex.com/UxYBwLh2g4e